taskdb/src/lib.rs

```diff
@@ -456,11 +458,12 @@ mod test {
         };
         let mut task_manager = get_task_manager(&opts);
 
+        let block_id = rand::thread_rng().gen_range(0..1000000);
```

Comment:
// This is an off-topic bug fix, but it prevent the PR passing CI.

The `mem_db` is static and will be shared across multiple test cases. Additionally, the test cases `test_new_taskmanager` and `test_enqueue_twice` use the same task descriptor, which could potentially lead to collisions. To address this, I have modified the task descriptor to prevent any such collision issues.

Reply:
maybe no need to use random but a fixed & non-conflict id, anyway, I am ok with this.